### PR TITLE
te: v0.1.2

### PR DESCRIPTION
**Pull request by jetaggart**

### CODE REVIEW

File: Library/Formula/te.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+require "language/haskell"
+
+class Te < Formula
+  include Language::Haskell::Cabal
+
+  homepage "https://github.com/jetaggart/te"
+  url "https://github.com/jetaggart/te/archive/v0.1.1.tar.gz"
+  version "0.1.0"
```
Comment (tdsmith) [inline Library/Formula/te.rb]: This should be discovered fine from the URL

**Comment by jetaggart** [comment]
Any idea why this is failing on mavericks and mountain lion? It doesn't look like it's related to the formula itself. It looks like git checkout issues.

http://bot.brew.sh/job/Homebrew%20Pull%20Requests/21807/version=mavericks/console

**Comment by tdsmith** [comment]
Might be superstitious of me but want to squash these into a single commit and see if it continues to fail? We'll want it squashed eventually anyway.

File: Library/Formula/te.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  def install
+    cabal_sandbox do
+      cabal_install "--only-dependencies"
+      cabal_install "--prefix=#{prefix}"
+    end
+    cabal_clean_lib
+  end
+
+  test do
+    system "te", "help"
```
Comment (mikemcquaid) [inline Library/Formula/te.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

**Comment by mikemcquaid** [comment]
Passing on this as it's a self-submission of a two day project with no stars or forks. I recommend creating your own tap. Sorry and thanks for the submission!

**Comment by jetaggart** [comment]
No worries, I read that part of the contributing docs late, but figured I'd get through the test failures. I'll try again if it gains some traction. Sorry for wasting any time!

**Comment by mikemcquaid** [comment]
No worries, thanks!

**Pull request closed without merging**